src/components/elements/default-tab.js

```diff
@@ -45,20 +45,20 @@ const TabButton = styled.button`
 const TabList = styled.div`
     display: flex;
     width: 100%;
-    justify-content: center;
+    justify-content: ${(props) => (props.type == 'blog-featured' ? 'start' : 'center')};
     position: relative;
     overflow: auto;
 
-    @media ${device.mobileL} {
-        justify-content: space-between;
+    @media ${device.tablet} {
+        justify-content: ${(props) => (props.type == 'blog-featured' ? 'center' : 'space-between')};
     }
```

Comment:
This is gonna affect other pages globally as well. If you want the flexibility, I suggest that for desktop you pass in prop `jc` and set fallback to `center`, and for mobile/tablet you can pass in `mobile_jc` or `tablet_jc` with original values as fallback.

Reply:
I updated the current code to make it doesn't affect other page , pages those are using this component are : `market page` and `dmt5signals` page

Do you think I can keep with my current change or implement your suggestion?

Reply:
It works for just this use case. But it will not be reusable if another design requires it to be jc flex-end. We would then need to refactor it again the next round. I do recommend allowing the props to be passed in so that the person can set the alignment him/herself.

Reply:
okay I alrd updated it, please have a look